.htmltest.external.yml

```diff
@@ -1,3 +1,6 @@
 DirectoryPath: public
 IgnoreDirectoryMissingTrailingSlash: true
 IgnoreAltMissing: true
+IgnoreEmptyHref: true
```

Comment:
Really, why?

Reply:
with the link checker we are using there wound up being a ton of href blank errors, like this:

```
docs/v2.3/api/index.html
  href blank --- docs/v2.3/api/index.html -->
```

but I wasn't able to track down what the _actual_ blank href was to correct it, so i turned off that particular check.

With `IgnoreEmptyHref: true` removed, our log looks like this:

```
➜  etcd-io-website git:(87-links-updates) make check-links
npm run production-build

> preproduction-build
> npm run get-submodules && ./check-hugo.sh


> get-submodules
> git submodule update --init --recursive --depth 1


> production-build
> hugo --cleanDestinationDir --minify

Start building sites …

                   | EN
-------------------+------
  Pages            | 451
  Paginator pages  |   0
  Non-page files   | 170
  Static files     |  32
  Processed images |   0
  Aliases          |   0
  Sitemaps         |   1
  Cleaned          |   0

Total in 2185 ms
# https://wjdp.uk/work/htmltest/
curl https://htmltest.wjdp.uk | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10278  100 10278    0     0  96056      0 --:--:-- --:--:-- --:--:-- 96056
wjdp/htmltest info checking GitHub for latest tag
wjdp/htmltest info found version: 0.14.0 for v0.14.0/linux/amd64
wjdp/htmltest info installed ./bin/htmltest
bin/htmltest
htmltest started at 04:52:02 on public
========================================================================
docs/v2.3/api/index.html
  href blank --- docs/v2.3/api/index.html -->
docs/v2.3/production-users/index.html
  href blank --- docs/v2.3/production-users/index.html -->
docs/v2.3/backward_compatibility/index.html
  href blank --- docs/v2.3/backward_compatibility/index.html -->
docs/v2.3/admin_guide/index.html
  href blank --- docs/v2.3/admin_guide/index.html -->
docs/v2.3/04_to_2_snapshot_migration/index.html
  href blank --- docs/v2.3/04_to_2_snapshot_migration/index.html -->
docs/v2.3/docker_guide/index.html
  href blank --- docs/v2.3/docker_guide/index.html -->
docs/v2.3/upgrade_2_1/index.html
  href blank --- docs/v2.3/upgrade_2_1/index.html -->
docs/v2.3/auth_api/index.html
  href blank --- docs/v2.3/auth_api/index.html -->
docs/v2.3/upgrade_2_2/index.html
  href blank --- docs/v2.3/upgrade_2_2/index.html -->
docs/v2.3/runtime-reconf-design/index.html
  href blank --- docs/v2.3/runtime-reconf-design/index.html -->
docs/v2.3/upgrade_2_3/index.html
  href blank --- docs/v2.3/upgrade_2_3/index.html -->
docs/v2.3/authentication/index.html
  href blank --- docs/v2.3/authentication/index.html -->
docs/v2.3/discovery_protocol/index.html
  href blank --- docs/v2.3/discovery_protocol/index.html -->
docs/v2.3/members_api/index.html
  href blank --- docs/v2.3/members_api/index.html -->
docs/v2.3/libraries-and-tools/index.html
  href blank --- docs/v2.3/libraries-and-tools/index.html -->
docs/v2.3/glossary/index.html
  href blank --- docs/v2.3/glossary/index.html -->
docs/v2.3/configuration/index.html
  href blank --- docs/v2.3/configuration/index.html -->
docs/v2.3/proxy/index.html
  href blank --- docs/v2.3/proxy/index.html -->
docs/v2.3/errorcode/index.html
  href blank --- docs/v2.3/errorcode/index.html -->
docs/v2.3/clustering/index.html
  href blank --- docs/v2.3/clustering/index.html -->
docs/v2.3/internal-protocol-versioning/index.html
  href blank --- docs/v2.3/internal-protocol-versioning/index.html -->
docs/v2.3/other_apis/index.html
  href blank --- docs/v2.3/other_apis/index.html -->
docs/v2.3/dev/release/index.html
  href blank --- docs/v2.3/dev/release/index.html -->
docs/v2.3/dev/index.html
  href blank --- docs/v2.3/dev/index.html -->
docs/v2.3/security/index.html
  href blank --- docs/v2.3/security/index.html -->
docs/v2.3/runtime-configuration/index.html
  href blank --- docs/v2.3/runtime-configuration/index.html -->
docs/v3.3/dl_build/index.html
  href blank --- docs/v3.3/dl_build/index.html -->
docs/v3.3/production-users/index.html
  href blank --- docs/v3.3/production-users/index.html -->
docs/v3.3/rfc/index.html
  href blank --- docs/v3.3/rfc/index.html -->
docs/v3.3/learning/learner/index.html
  href blank --- docs/v3.3/learning/learner/index.html -->
docs/v3.3/learning/auth_design/index.html
  href blank --- docs/v3.3/learning/auth_design/index.html -->
docs/v3.3/learning/client-architecture/index.html
  href blank --- docs/v3.3/learning/client-architecture/index.html -->
docs/v3.3/learning/client-feature-matrix/index.html
  href blank --- docs/v3.3/learning/client-feature-matrix/index.html -->
docs/v3.1/dl_build/index.html
  href blank --- docs/v3.1/dl_build/index.html -->
docs/v3.1/production-users/index.html
  href blank --- docs/v3.1/production-users/index.html -->
docs/v3.1/rfc/index.html
  href blank --- docs/v3.1/rfc/index.html -->
docs/v3.1/libraries-and-tools/index.html
  href blank --- docs/v3.1/libraries-and-tools/index.html -->
docs/v3.1/dev-internal/index.html
  href blank --- docs/v3.1/dev-internal/index.html -->
docs/v3.1/docs/index.html
  href blank --- docs/v3.1/docs/index.html -->
docs/index.html
  target does not exist --- docs/index.html --> /docs/_index.md
docs/v3.2/dl_build/index.html
  href blank --- docs/v3.2/dl_build/index.html -->
docs/v3.2/production-users/index.html
  href blank --- docs/v3.2/production-users/index.html -->
docs/v3.2/rfc/index.html
  href blank --- docs/v3.2/rfc/index.html -->
docs/v3.2/learning/auth_design/index.html
  href blank --- docs/v3.2/learning/auth_design/index.html -->
docs/v3.2/dev-internal/index.html
  href blank --- docs/v3.2/dev-internal/index.html -->
docs/v3.2/docs/index.html
  href blank --- docs/v3.2/docs/index.html -->
========================================================================
✘✘✘ failed in 629.81541ms
46 errors in 403 documents
make: *** [Makefile:15: run-link-checker] Error 1
```

Reply:
Let's track this via #180